package.json

```diff
@@ -26,7 +26,10 @@
     "workspace-generator": "nx workspace-generator",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "proxy": "node tools/proxy/proxy.js"
+    "proxy": "node tools/proxy/proxy.js",
```

Comment:
The following 3 scripts will make it easier to (locally) verify the health of the project.